programs/jslib.cpp

```diff
@@ -6,50 +6,110 @@
 
 #include "emp-tool/io/i_raw_io.h"
 #include "emp-ag2pc/2pc.h"
+#include "emp-agmpc/mpc.h"
 
-void run_impl(int party);
+void run_2pc_impl(int party, int nP);
+void run_mpc_impl(int party, int nP);
 
 // Implement send_js function to send data from C++ to JavaScript
-EM_JS(void, send_js, (const void* data, size_t len), {
+EM_JS(void, send_js, (int party2, char channel_label, const void* data, size_t len), {
```

Comment:
Why is it named `party2`?

Reply:
Oh yeah that's carried over from the naming in agmpc:
https://github.com/emp-toolkit/emp-agmpc/blob/0add81e/emp-agmpc/abitmp.h#L80

However, yeah it's confusing. For our code I've now replaced `party2` with `toParty`/`fromParty`/`otherParty` depending on the context.